lib/converters/index.js

```diff
@@ -14,11 +15,11 @@ function generateOutputData(jsonData, outputExtension) {
       return vtt(jsonData);
     default:
       throw Error(`File type ${outputExtension} is not supported. Supported output file types include:\n`
-      + 'srt');
+      + '\'.srt\'\n\'.vtt\'');
```

Comment:
You can combine this into one line `...include: .srt, .vtt`  I think the newlines are unnecessary.

Reply:
Agreed